### PR TITLE
python3-rdflib: update, remove python-rdflib; python3-isodate: update, remove python-isodate; remove python-notify

### PR DESCRIPTION
**Pull request by classabbyamp**
- python3-rdflib: update to 6.2.0, remove python-rdflib.
- python-notify: remove package

#### Testing the changes
- I tested the changes in this PR: **YES**

### CODE REVIEW

File: srcpkgs/python3-rdflib/patches/setup.patch [hunk 1]
```diff
@@ -0,0 +1,9 @@
+--- a/setup.py
++++ b/setup.py
+@@ -88,5 +88,5 @@
+             "rdfgraphisomorphism = rdflib.tools.graphisomorphism:main",
+         ],
+     },
+-    **kwargs,
++    **kwargs
```
Comment (ahesford) [inline srcpkgs/python3-rdflib/patches/setup.patch:8]: Why is this change necessary? The interpreter should have no problem with `**kwargs,` here.

Comment (classabbyamp) [reply srcpkgs/python3-rdflib/patches/setup.patch:8]: not sure, it `SyntaxError`s

Comment (ahesford) [reply srcpkgs/python3-rdflib/patches/setup.patch:8]: Not for me...

Comment (classabbyamp) [reply srcpkgs/python3-rdflib/patches/setup.patch:8]: weird, it was happening for me on multiple versions when I made that.
